assignments/array-methods.js

```diff
@@ -56,28 +56,58 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach((item) => fullName.push(item.first_name +' '+ item.last_name));
```

Comment:
this is an awesome time to try out those string template literals!
```suggestion
runners.forEach((item) => fullName.push(`${item.first_name} ${item.last_name}`));
```